blog/tech-talk-window-resize-behavior.md

```diff
@@ -0,0 +1,290 @@
+---
+title: 'Tech Talk #1: Improving Window Resize Behavior'
+date: 2025-12-19T00:00:00.000Z
+authors:
+  - name: nikwen
+    url: 'https://github.com/nikwen'
+    image_url: 'https://github.com/nikwen.png?size=96'
+slug: tech-talk-window-resize-behavior
+tags: [techtalk, internals]
+---
+
+_We're launching a new blog post series where we share glimpses into our work on Electron. If you find this work interesting, please consider [contributing](https://github.com/electron/electron/)!_
+
+---
+
+Recently, I worked on improving Electron and Chromium's window resize behavior.
+
+## The bug
+
+We were seeing an issue on Windows where old frames would become visible while resizing a window:
+
+![Animated GIF showing the issue where old frames would be shown while resizing windows](/assets/img/blog/tech-talk-window-resize-behavior/window-resize-bug-showing-old-frames.gif)
+
+## What made this bug particularly interesting?
+
+1. It was challenging.
```

Comment:
Expand on this a bit? e.g., it was challenging because it involved going deep into GPU timing code in Chromium.